src/graphql/graphql.schema.ts

```diff
@@ -92,11 +95,19 @@ const schema = gql`
     pageInfo: PageInfo!
   }
 
+  type MapSearchResult implements SearchResult {
+    totalCount: Int!
+    results: [CombinedMapResult!]!
+    filters: [Filter!]
+    pageInfo: PageInfo!
+  }
+
   type Filter {
     type: String!
     label: String!
-    options: [FilterOptions!]
-    filterType: String
+    options: [FilterOptions!]!
+    # TODO: Make filterType a proper enum.
```

Comment:
Do we need a new issue for this?

Reply:
Or can you use the already exisiting `FilterType` for this?

Reply:
I added the enum to the schema, and replaced the existing FilterType instances